.github/workflows/e2e-nvidia-l40s-x4.yml

```diff
@@ -23,6 +23,18 @@ jobs:
       label: ${{ steps.start-ec2-runner.outputs.label }}
       ec2-instance-id: ${{ steps.start-ec2-runner.outputs.ec2-instance-id }}
     steps:
+      - name: Checkout "free-disk-space" in-house CI action
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          repository: instructlab/ci-actions
+          path: ci-actions  # clone the "ci-actions" repo to a local directory called "ci-actions", instead of overwriting the current dir contents
+          ref: free-disk-space  # temporary branch with my changes
```

Comment:
https://github.com/instructlab/ci-actions/pull/2 is merged, so you can remove this temporary branch reference here.

Reply:
Thanks. I'll update this PR once we cut a release in that repo! I want to pin to a release.